src/main/java/fact/datacorrection/PatchJumpRemoval.java

```diff
@@ -38,6 +38,9 @@
     @Parameter(required = true)
     public String startCellKey = null;
 
+    @Parameter(required = false, description = "The key containing the UnixTimeUTC")
+    public String unixTimeKey = "UnixTimeUTC";
```

Comment:
Why is this a parameter?

Reply:
I use this processor twice for different inputs with different times one will use 'UnixTimeUTC' the other 'LONS_UnixTimeUTC'. This is due to the fact that i have to calibrate the pedestal that I superimpose.